.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -19,34 +19,33 @@ concurrency:
 jobs:
   build-new-node:
     name: Build node and runtime artifacts (PR version)
-    uses: Cardinal-Cryptography/aleph-node/.github/workflows/build-node-and-runtime.yml@main
-
+    uses: ./.github/workflows/build-node-and-runtime.yml
```

Comment:
+1

I'm quite sure that at the time of introducing reusable workflows to our repo, the only way of pointing workflow was through full git path :(